databricks-geococoding/README.md

```diff
@@ -0,0 +1,41 @@
+![Precisely](../Precisely_Logo.png)
+
+# Databricks Geocoding Sample
+This sample for Databricks demonstrates how to install, configure, and execute against the appropriate geocoding spark driver. 
+After cloning or downloading the repo you will have 2 html files to be imported into your workspace https://docs.databricks.com/notebooks/notebooks-manage.html.
+
+## Data
+This sample does not include the data, but does show how to integrate with the Precisely Data Experience.  Using your credentials you will be assured to be using the most recent data vintage.  Also this expidites the setting up of data on your cluster.
+
+* **Geocoding Reference Data**: USA-MasterLocationData, USA-TomTom. 
+
+## Setting up DataExperience integration
+  1) Collect your Data Experience API Key and Secret Key by visiting https://data.precisely.com/autodownload  You will need these for the Geocoding Installation Workspace.
+  2) The DataExperience SDK has been prebuilt and included in the repo at /lib/precisely-bigdata-pdx-sdk2.7-full.jar.  It will need to be added to your Databricks FileStore https://docs.databricks.com/data/filestore.html.  You will need to set the location of the file in Configuration and Variables cell of Geocode Installation workbook.
+
+## Building the sample
+The sample is broken into 2 workspaces.  One for installing the other for configuring and executing Geocodes.
+
+## Installing Geocoding
+The installation is split up between multiple cells in the Geocoding Installation Workspace.  For best results you should execute each cell individually in top down order.  
```

Comment:
cells -> "commands", workspace -> "notebook" to keep with databricks terms.

The results should be the same as long as everything is configured correctly... Perhaps something like: After updating the configuration section, you can execute each cell individually or run the entire notebook to install and configure the geocoding libraries and reference data.

Reply:
How about this, covering this line (above) and the next (below):
The Geocoding Installation notebook contains the commands necessary to install the geocoding libraries and reference data.  The first command provides configuration variables for the notebook and will need to be updated with values specific to your environment.  After updating the configuration section, you can execute each cell individually or run the entire notebook to perform the installation.

Reply:
done